test/templates/fake_middleware.rb

```diff
@@ -0,0 +1,50 @@
+require "action_view"
+require "pathname"
+
+# Fake rack app to render templates
+class FakeMiddleware
+  def call(env)
+    [200, header, [render(req_path(env))]]
+  end
+
+  private
+
+  # default header
+  def header
+    {"Content-Type" => "application/javascript"}
+  end
+
+  # extract target path from REQUEST_PATH
+  def req_path(env)
+    env["REQUEST_PATH"].match(req_path_regex)[1]
+  end
+
+  def render(template)
+    view.render(template: template, layout: nil)
+  end
+
+  def view
+    @view ||= create_view
+  end
+
+  def create_view
+    lookup_context = ActionView::LookupContext.new(view_path)
+    lookup_context.cache = false
+    FakeView.new(lookup_context)
+  end
+
+  class FakeView < ActionView::Base
+    def render_inlined_string(template)
+      render(template: template, layout: "layouts/inlined_string")
+    end
+
+    # helper for test runners
+    def glob(glob_path)
+      root_path = Pathname.new(Helper.test_root)
```

Comment:
Right now this class knows about the Helper. Can we drop this dependency somehow?